docs/en/observability/logs-parse-filter.asciidoc

```diff
@@ -0,0 +1,1227 @@
+[[logs-parse-filter]]
+= Parse and explore logs
+
+Parsing your log data takes unstructured or semi-structured logs and breaks them into meaningful fields. You can use those fields to explore and analyze your data. For example, you can find logs in a specific timestamp range or filter logs by log level to focus on potential issues. 
+
+See the following sections for more on parsing and exploring your logs:
+
+* <<logs-stream-parse>> – extracting structured fields from your unstructured log data.
+* <<logs-stream-reroute>> – route data from the generic data stream to a target data stream
+* <<logs-filter>> – Filter your data to focus on logs that meet specific criteria
+* <<logs-aggregate>> – Analyze and summarize your log data to find patterns and gain insight.
+
+[discrete]
+[[logs-stream-parse]]
+= Parse your logs
+
+Make your logs more useful by extracting structured fields from your unstructured log data. Extracting structured fields makes it easier to search, analyze, and filter your log data. 
+
+Let's look at this log example:
+
+[source,log]
+----
+2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.
+----
+
+From the *Dev Tools* console found in {ecloud} in the navigation menu under *Management*, add the example log to {es} using the following command:
+
+[source,console]
+----
+POST logs-example-default/_doc
+{
+  "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%."
+}
+----
+
+The previous command stores the document in the `logs-example-default` data stream. Retrieve it with the following search:
+
+[source,console]
+----
+GET /logs-example-default/_search
+----
+
+The results should look like this:
+
+[source,JSON]
+----
+{
+  ...
+  "hits": {
+    ...
+    "hits": [
+      {
+        "_index": ".ds-logs-example-default-2023.08.09-000001",
+        ...
+        "_source": {
+          "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.",
+          "@timestamp": "2023-08-09T17:19:27.73312243Z"
+        }
+      }
+    ]
+  }
+}
+----
+
+{es} indexes the `message` field by default meaning you can search for phrases like `WARN` or `Disk usage exceeds`. For example, the following command searches for the phrase `WARN` in the log `message` field:
```

Comment:
Lets mention that the timestamp was automatically added as it was not set and is set to `now`.